tests/e2e/specs/combine/import-scenarios.js

```diff
@@ -33,4 +35,16 @@ describe('Combine: Export and import combined scenarios', () => {
           .parentsUntil(selector('layer-panel'))
       })
   })
+
+  it('Imports scenario and deselects a location', () => {
+    cy.url()
+      .should('contain', '/combine/7/19422,19428', { timeout: 30000 })
+      .then(() => {
+        cy.wait(5000)
+
+        cy.get('.leaflet-marker-icon')
+          .eq(1)
+          .click({ force: true })
```

Comment:
Can we add a check to see if the actual marker image changed?

Reply:
Changed.